make.js

```diff
@@ -560,6 +564,16 @@ target.bump = function() {
         }
 
         taskJson.version.Patch = taskJson.version.Patch + 1;
+        taskLocJson.version.Patch = taskLocJson.version.Patch + 1;
```

Comment:
Can we just do `taskLocJson.version.Patch = taskJson.version.Patch` here and not do the verification? It seems like that will always be the right behavior.

Reply:
I was debating that but then that would indicate a user error from not bumping both so I wanted to be aware of it explicitly.